var/spack/repos/builtin/packages/ncbi-rmblastn/package.py

```diff
@@ -26,11 +26,102 @@
 
 
 class NcbiRmblastn(AutotoolsPackage):
-    """RMBlast search engine for NCBI"""
+    """RMBlast search engine for NCBI."""
 
     homepage = "https://www.ncbi.nlm.nih.gov/"
     url      = "ftp://ftp.ncbi.nlm.nih.gov/blast/executables/rmblast/LATEST/ncbi-rmblastn-2.2.28-src.tar.gz"
 
     version('2.2.28', 'fb5f4e2e02ffcb1b17af2e9f206c5c22')
+    version('2.6.0', 'c8ce8055b10c4d774d995f88c7cc6225',
+            url='ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.6.0/ncbi-blast-2.6.0+-src.tar.gz')
```

Comment:
why did they change the directory of the download? And will future version use this directory? If so, it would be better to put the manual set to 2.2.28

Reply:
Because this version is really just a patch to the regular ncbi-blastn. I have no idea if they're going to keep up this style of distribution with later versions (assuming there are any).